code/src/components/ProductIngredients.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components/macro'
+
+const ContentWrapper = styled.div`
+  display: flex; 
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 20px 20px 0 20px;

```

Comment:
if this styling is used in many files, you could create a common Wrapper component and import that in all the files where you want it. You could then add extra styling to it in specific files. 
Like:
```
const Wrapper = styled.div`
  display: flex; 
  justify-content: center;
  align-items: center;
  width: 100%;
  padding: 20px 20px 0 20px;
`
// In one file, and then something like:
import {Wrapper} from 'components/common'

const IngredientWrapper = styled(Wrapper)`
  extra styling here...
`
```